integrations/github-cloud/github_integration/models/__init__.py

```
@@ -0,0 +1,5 @@
+from .issue import *
```

Ruff (0.11.9)

`from .issue import *` used; unable to detect undefined names

(F403)